ko.po: warning: Charset "EUC-KR" is not supported. gmsgfmt relies on iconv(),
                and iconv() does not support "EUC-KR".
                Installing GNU libiconv and then reinstalling GNU gettext
                would fix this problem.
                Continuing anyway.

Same with zh_CN.

--- vim-8.1.1713/src/po/Make_all.mak.~1~	2019-07-18 14:46:42.000000000 +0000
+++ vim-8.1.1713/src/po/Make_all.mak	2019-07-20 07:36:27.787001577 +0000
@@ -38,7 +38,6 @@
 		uk \
 		uk.cp1251 \
 		vi \
-		zh_CN \
 		zh_CN.UTF-8 \
 		zh_CN.cp936 \
 		zh_TW \
@@ -81,7 +80,6 @@
 		uk.po \
 		uk.cp1251.po \
 		vi.po \
-		zh_CN.po \
 		zh_CN.UTF-8.po \
 		zh_CN.cp936.po \
 		zh_TW.po \
@@ -117,7 +115,6 @@
 		uk.mo \
 		vi.mo \
 		zh_CN.UTF-8.mo \
-		zh_CN.mo \
 		zh_TW.UTF-8.mo \
 		zh_TW.mo \
 
@@ -171,7 +168,6 @@
 		uk.cp1251.ck \
 		vi.ck \
 		zh_CN.UTF-8.ck \
-		zh_CN.ck \
 		zh_CN.cp936.ck \
 		zh_TW.UTF-8.ck \
 		zh_TW.ck \
